Xamarin.Forms.Platform.UAP/ListViewRenderer.cs

```diff
@@ -255,6 +260,20 @@ void UpdateGrouping()
 		}
 
 		void UpdateSelectionMode()
+		{
+			if (Element.SelectionMode == ListViewSelectionMode.None)
+			{
+				List.SelectionMode = Windows.UI.Xaml.Controls.ListViewSelectionMode.None;
+				List.SelectedIndex = -1;
```

Comment:
This should be on core no? so it can be reused for all platforms, like a internal ResetSelectedItem for something , for example iOS is only setting the SelectedItem to null and not the index

Reply:
`List` here is the native UWP `Windows.UI.Xaml.Controls.ListView` so this code doesn't generalize to other platforms.

In the iOS `ListViewRenderer.UpdateSelectionMode()` method we call `UITableView.DeselectRow()` which does the equivalent.

On Android there's no need to do anything as the `Xamarin.Forms.Platform.Android.ListViewAdapter` already listens to the `ListView.ItemSelected` event and when the Android `ListViewRenderer.UpdateSelectionMode()` method sets `SelectedItem` to `null` it will deselect the currently selected row in the UI.

Reply:
Oh Sorry, i was reading this as ListView Forms.